specs/phase0/p2p-interface.md

```diff
@@ -195,6 +198,22 @@ This section outlines constants that are used in this spec.
 | `ATTESTATION_PROPAGATION_SLOT_RANGE` | `32` | The maximum number of slots during which an attestation can be propagated. |
 | `MAXIMUM_GOSSIP_CLOCK_DISPARITY` | `500ms` | The maximum milliseconds of clock disparity assumed between honest nodes. |
 
+## MetaData
+
+Clients MUST locally store the following `MetaData`:
+
+```
+(
+  seq_number: uint64
+  attnets: Bitvector[ATTESTATION_SUBNET_COUNT]
+)
+```
+
+Where
+
+- `seq_number` is a `uint64` starting at `0` used to version the node's metadata. If any other field in the local `MetaData` changes, the node MUST increment `seq_number` by 1.
```

Comment:
Given the different contents, does that mean it is different than an ENR sequence number? Stating that it is different, and can be out of sync, would be good, since the names are the same.

Reply:
yeah, they can be. Will do so